index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div class="container">
+      <header class="header">
+        <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
```

Comment:
use hx tag for the semantic reason like h2,h3,..etc.

Reply:
well done by adding inline comments , but this comment is confusing as you are asking the student to use hx tag while they are already using them , please always give arguments on why a change is requested and what's wrong with the actual code or how the change can improve it if not wrong